examples/trainercontroller_configs/log_controller.yaml

```
@@ -0,0 +1,16 @@
+controller_metrics:
+  - name: state
+    class: TrainingState
+operations:
+  - name: logcontrolstep
+    class: LogControl
+    arguments:
+      log_format: 'This is a test log format [{event_name}] => {tc_metrics[state]}'
```

Shouldn't this be just `state`?
```suggestion
      log_format: 'This is a test log format [{event_name}] => {state}'
```

---

Addressed.